packages/manager/apps/veeam-backup/src/pages/listing/DatagridCell.component.tsx

```diff
@@ -24,17 +23,18 @@ import {
 } from '@ovh-ux/manager-module-vcd-api';
 import { vcdOrganizationAppName } from '@/veeam-backup.config';
 import { urls } from '@/routes/routes.constant';
+import TEST_IDS from '@/utils/testIds.constants';
 
 export const DisplayNameCell = (backup: VeeamBackup): JSX.Element => {
   const navigate = useNavigate();
   return (
     <DataGridTextCell>
-      <OsdsLink
-        color={ODS_THEME_COLOR_INTENT.primary}
+      <OdsLink
+        label={getVeeamBackupDisplayName(backup)}
         onClick={() => navigate(urls.dashboard.replace(':id', backup.id))}
-      >
-        {getVeeamBackupDisplayName(backup)}
-      </OsdsLink>
+        href={undefined}
```

Comment:
```suggestion
        href={urls.dashboard.replace(':id', backup.id)}
```

I don't known if this work well. But can be better for UX

Reply:
Using href will reload the whole page, not an expected behavior on a SPA